### PR TITLE
Bugfix/kad 3058

### PR DESCRIPTION
**Pull request by gilbert-hernandez**
🎫  #[Jira Ticket]
[https://stellarwp.atlassian.net/browse/KAD-3058](https://stellarwp.atlassian.net/browse/KAD-3058)
...

### Issue Info
- [x] Were you able to reproduce the issue?
- [ ] Is the issue from the ticket solved? (If not, why?)

### Checklist
- [ ] I have performed a self-review.
- [ ] No unrelated files are modified.
- [ ] No debugging statements exist (Ex: console.log, error_log).
- [ ] There are no warnings or notices in the wordpress error log.
- [ ] Passes all tests (linting, acceptance, & unit)

### Block specific checklist (where relevant)
- [ ] Tested with an existing instance of this block .
- [ ] Tested creating a new instance of this block.
- [ ] Tested with Dynamic content & Elements.

#### Are there dependent changes in another repository?
- [x] No.
- [ ] Yes. Please provide the link to the PR.

### CODE REVIEW

**Comment by gilbert-hernandez** [comment]
The padding controls now show "sm" if there is a BG and there aren't L/R padding set. The front end used the customizer content edge while the backend used the "sm" space. I updated the front end to use the "sm" spacing as well. Otherwise, I would need to add to the src/blocks/rowlayout/constants.js file to show something custom. 

Instead, now a background adds a "sm" L/R padding by default (for front and back), and the padding controls show it in the block settings.

**Comment by mark-c-woodard** [comment]
Good idea following some of the logic already on that component for placeholders top/bottom values.

Since the bg only applies left/right padding, we need to respect the values already in place for top/bottom though. So I updated the padding placeholders to use the set top/bottom values.

I also reverted the frontend change. I don't think we want to change people's frontends like that. 
Also, looking at the code surrounding this bg color padding in the background, it reads like a compromise. 
Like we actually want to set `--global-content-edge-padding` when a bg color is applied, but it's really inconvenient to apply that value in the backend. The spacing visualizers and other components can't really handle a value like `var(--global-content-edge-padding)` so I think `sm` was the closest thing that made everything play nice. Not exactly true to frontend, but a compromise.

So long story short I think we want to match the frontend. It's just too inconvenient, so `sm` is close enough.
